NeoMathEngine/src/MathEngineDnnConv.h

```diff
@@ -59,8 +121,15 @@ struct CCommon3dConvolutionDesc : public C3dConvolutionDesc {
 	int StrideWidth;
 	int StrideDepth;
 
-	CCommon3dConvolutionDesc( const CBlobDesc& source, const CBlobDesc& result, const CBlobDesc& filter,
-			int paddingHeight, int paddingWidth, int paddingDepth, int strideHeight, int strideWidth, int strideDepth ) :
+	enum { TSMMD_1x1x1_Forward, TSMMD_1x1x1_Backward, TSMMD_1x1x1_Learn,
```

Comment:
It looks like we can use only 3 values here (`Forward/Backward/Learn`) because convolution can't be usual and 1x1x1 at the same time.

Reply:
fixed